script/script_basics.md

```diff
@@ -0,0 +1,14 @@
+# Script Basics
```

Comment:
Great idea to add an intro page on bitcoin script. I was jumping into the deep end a bit too quickly there.